### PR TITLE
Add inventory item notifications

### PR DESCRIPTION
**Pull request by mzlarsson**
Add notifications on item add and remove

### CODE REVIEW

File: inventory/public/style.css [hunk 1]
```diff
@@ -83,6 +84,51 @@ body{
...
+}
+.inventory-notification.visible{
+	position:relative;
+	display:flex;
+	animation-name:inventory-notification;
+	animation-duration:5s;
+}
+@keyframes inventory-notification{
+	0% {opacity:0;}
+	20% {opacity:1;}
```
Comment (AntonNeld) [inline inventory/public/style.css]: I played around with it and I think 5% would look good here. It feels snappier, or what do you think?

**Review by AntonNeld — COMMENTED** [review]
When I tried this (with the full setup) the notification had a transparent background. Does it work fine for you?

I didn't check out the corresponding Core commit, but that shouldn't affect this, right?

**Comment by mzlarsson** [comment]
> When I tried this (with the full setup) the notification had a transparent background. Does it work fine for you?
> 
> I didn't check out the corresponding Core commit, but that shouldn't affect this, right?

My bad, forgot to update the drive with new data for `inventory-data.zip`. The code uses a theme resource for item backgrounds to be able to customize inventory looks. File is now uploaded!

File: inventory/public/style.css (hunk 1 above)
Comment (mzlarsson) [reply inventory/public/style.css]: Haven't played around so much with it, but 5% sounds nice!

**Review by AntonNeld — APPROVED** [review]
My bad too, then :) I didn't think to update it anyway.

**Pull request merged**